### PR TITLE
feat(demos): align images used in benchmark to stride 64

### PR DESCRIPTION
**Pull request by cristian-stoica**
### Description of the feature or fix

A clear and concise description of what the bug or new feature is.

### Notes
- Update the [Documentation](https://github.com/lvgl/lvgl/tree/master/docs) if needed.
- Add [Examples](https://github.com/lvgl/lvgl/tree/master/examples) if relevant.
- Add [Tests](https://github.com/lvgl/lvgl/blob/master/tests/README.md) if applicable.
- If you added new options to `lv_conf_template.h` run [lv_conf_internal_gen.py](https://github.com/lvgl/lvgl/blob/master/scripts/lv_conf_internal_gen.py) and update [Kconfig](https://github.com/lvgl/lvgl/blob/master/Kconfig).
- [x] Run `scripts/code-format.py` ([astyle](http://astyle.sourceforge.net/install.html) version [v3.4.10](https://github.com/szepeviktor/astyle/releases/tag/v3.4.10) needs to be installed) and follow the [Code Conventions](https://docs.lvgl.io/master/CODING_STYLE.html).
- Mark the Pull request as [Draft](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request) while you are working on the first version, and mark is as _Ready_ when it's ready for review.
- When changes were requested, [re-request review](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review) to notify the maintainers.
- Help us to review this Pull Request! Anyone can [approve or request changes](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/approving-a-pull-request-with-required-reviews).

### CODE REVIEW

**Review by XuNeo — DISMISSED** [review]

File: demos/benchmark/assets/img_benchmark_cogwheel_argb.c [hunk 1]
```diff
@@ -1,434 +1,136 @@
...
 #if LV_USE_DEMO_BENCHMARK
-
 #ifndef LV_ATTRIBUTE_MEM_ALIGN
-    #define LV_ATTRIBUTE_MEM_ALIGN
+#define LV_ATTRIBUTE_MEM_ALIGN
 #endif
 
-#ifndef LV_ATTRIBUTE_IMAGE_IMG_COGWHEEL_ARGB
-    #define LV_ATTRIBUTE_IMAGE_IMG_COGWHEEL_ARGB
+#ifndef LV_ATTRIBUTE_IMG_DUST
```
Comment (kisvegabor) [inline demos/benchmark/assets/img_benchmark_cogwheel_argb.c]: Why is it called `DUST`?

Comment (nicusorcitu) [reply demos/benchmark/assets/img_benchmark_cogwheel_argb.c]: @anaGrad I see all the files have been changed with this define.
I suppose this is a python script effect, please have it fixed all over.
Thanks.

Comment (anaGrad) [reply demos/benchmark/assets/img_benchmark_cogwheel_argb.c]: Is there any convention on the naming of the images? 

- I notice there is an inconsistency in the names (for example, in the assets of widgets, some images have widget in the name, others don't). 
image

- The names of the .c files is different from that of the .png file (for example in benchmark/assets). 
image

- There are multiple pngs with different names in different folders for what looks like the same image and I was wondering why is it like that (example: lvgl logo in demo/render vs demo/widgets) 
image

Comment (XuNeo) [reply demos/benchmark/assets/img_benchmark_cogwheel_argb.c]: I don't understand neither. It's used in all old .c images and I simply copied the original headers as template to python script.

Comment (kisvegabor) [reply demos/benchmark/assets/img_benchmark_cogwheel_argb.c]: The name of the image should be used in the ATTRIBUTE name. See [this](https://github.com/lvgl/lvgl/blob/release/v8.3/demos/widgets/assets/img_demo_widgets_avatar.c#L10) from v8.3.

The names are really not consistent now. We can work on that too in an other PR.

Comment (nicusorcitu) [reply demos/benchmark/assets/img_benchmark_cogwheel_argb.c]: Yes, that ATTRIBUTE has been fixed.

We will address separately the naming files .png vs .c It really has to be a consistency and avoid duplication.

**Comment by nicusorcitu** [comment]
@XuNeo What is the endians rule for this script for index formats?
https://github.com/lvgl/lvgl/blob/master/scripts/LVGLImage.py

As a matter of fact, LVGL is all over little endian fashion, right? So it should be the script acting.

**Comment by XuNeo** [comment]
Yes. Little endian is used.


https://github.com/lvgl/lvgl/blob/master/scripts/LVGLImage.py#L22-L39

**Comment by nicusorcitu** [comment]
My question is inside the byte, what is the order for index format type?
The VGLite can have these:
image

**Comment by XuNeo** [comment]
For indexed image, it's also little endian, differ from early VG-Lite can do. The highest bit is the first pixel.

https://github.com/lvgl/lvgl/blob/03498a2f83d36dfe7bd009a4451083a4f64ef0ba/src/libs/bin_decoder/lv_bin_decoder.c#L1035-L1043

The LVGLImage.py uses `pypng`, it outputs the exact same format.

**Comment by nicusorcitu** [comment]
OK, so according to this picture the python script generate a VG_LITE_INDEX_ENDIAN_BIG_ENDIAN.
318411835-4d56540b-0caf-42e6-8960-99149ef0d755

**Comment by nicusorcitu** [comment]
The endianes format for index format can be addressed later, if everything is fine with the c array files, you can merge them to master.
Thanks.

**Review by XuNeo — APPROVED** [review]
Have you modified the python script to generate correct macros using image file name? Please upload it too.

**Comment by nicusorcitu** [comment]
There was no python script changes. Just the new images aligned.

**Review by kisvegabor — APPROVED** [review]

**Pull request merged**